src/pineko/evolve.py

```diff
@@ -149,8 +158,10 @@ def evolve_grid(
     comparison_pdf : None or str
         if given, a comparison table (with / without evolution) will be printed
     """
-    x_grid, _pids, mur2_grid, _muf2_grid = grid.axes()
-    sv_method = None
```

Comment:
Why did you remove this?

Reply:
afaics it's not used anywhere in that function